CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- JSON API: `txprepare` now uses `output` as parameter other than `destination` and `satoshi`
```

Comment:
this should be called 'outputs', to denote the fact that the expected input is an array

Reply:
Yes, `outputs` is better.

Reply:
Fixed.

Reply:
Should we fix this CHANGELOG line too?

Reply:
I forgot it ;-(. Now I fixed.